release_notes/release_notes.stex

```diff
@@ -2873,6 +2873,12 @@ An incorrect approach to finding the current executable's path on
 NetBSD meant that boot files could not be found relative to the
 executable.
 
+\subsection{Prevent \scheme{path-build} from doing invalid memory accesses (10.2.0)}
+
+\scheme{path-build} mades invalid memory refences when first argument
+(\scheme{dir-path}) is the empty string.
+It now instead returns the second argument unmodified (\scheme{path}).
+
```

Comment:
```
\subsection{Invalid memory access in \scheme{path-build} (10.2.0)}

When the \scheme{dir-path} argument is the empty string, the \scheme{path-build}
procedure now returns the \scheme{path} argument instead of performing an invalid memory access.

Reply:
This wording is much better indeed. Took it.